.nf-core.yml

```diff
@@ -4,6 +4,7 @@ lint:
     - params.input
   files_unchanged:
     - .gitignore
+    - docs/images/nf-core-mcmicro_logo_dark.png
```

Comment:
I think the problem here is that a locally-generated version of this png got committed. Remember the fonts are rendered with slightly different subpixel positioning on GHA for some reason, so we just need to retain the original file that passes linting there. If you restore the original file you should be able to delete this line.